cluster/install/fluo-cluster/bin/fluo-cluster

```diff
@@ -58,6 +61,7 @@ ready)
   echo "  start        Start all services on cluster"
   echo "  kill         Kill cluster processes"
   echo "  ready        Checks if cluster is running and ready"
+  echo "  ashell       Runs Accumulo shell"
```

Comment:
added in ccb18a34769.  I like starting the accumulo shell using fluo-cluster as fluo-cluster has a lot of environment already set in `env.sh`.  If I put the command in the bashrc, I have put accumulo's password there too.